cve_bin_tool/format_checkers.py

```diff
@@ -76,9 +78,37 @@ def update_checker_table(file_path, markdown):
         f.writelines(lines)
 
 
+def update_allowed_words(checkers_array):
+    """Updates the allow.txt file with the new checkers"""
+    file_path = os.path.join(
+        os.path.abspath("."), ".github", "actions", "spelling", "allow.txt"
+    )
+
+    checkers_words = re.findall(r"[^0-9_]+", "_".join(checkers_array))
+    fileObj = open(file_path, "r")
+    words = fileObj.read().splitlines()
+    fileObj.close()
+
+    dict = requests.get(
+        "https://raw.githubusercontent.com/check-spelling/check-spelling/dictionary/dict.txt"
```

Comment:
Honestly, I hate making another network request here even to github because I feel like it increases the risk of something going wrong.  I wonder if we'd be better off just not bothering to check to see if the new checker is an existing dictionary word?

Reply:
Maybe we can ask the user if they want to check the online dictionary and make the network request only when they say yes?

Reply:
> Maybe we can ask the user if they want to check the online dictionary and make the network request only when they say yes?

The script is ran as a pre-commit script and it may not be possible for user confirmation. And adding words from the online dictionary to the allow list won't cause any issues in spell checking (so we could avoid looking up the online dictionary). Plus checker names are mostly not available in the online dictionary.